pandas/core/frame.py

```diff
@@ -2038,6 +2040,14 @@ def to_stata(
 
             .. versionadded:: 0.23.0
 
+        compression : {'infer', 'gzip', 'bz2', 'zip', 'xz', None}, default 'infer'
+            For on-the-fly compression of the output dta. If 'infer', then use
+            gzip, bz2, zip or xz if path_or_buf is a string ending in
+            '.gz', '.bz2', '.zip', or 'xz', respectively, and no compression
+            otherwise.
+
+            .. versionadded:: 1.2.0
```

Comment:
is this intentional? is it ok to target 1.1? maybe still a couple of weeks away.

Reply:
I wasn't sure of the timing; I've moved to 1.1.0.